packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -840,8 +840,8 @@ Common Environment Variables
     $diskInfo = Get-CimInstance -ClassName Win32_LogicalDisk | Out-String
     $psInfo = $PSVersionTable.PSVersion
     $psInfoClr = $PSVersionTable.CLRVersion
-    $chocoInfo = chocolatey --version
-    $installedPackages = chocolatey list -l -r -all
+    $chocoInfo = choco --version
+    $installedPackages = choco list -l -r -all
```

Comment:
```suggestion
    $installedPackages = choco list -r -all
```

There was another change in Chocolatey v2.0.0 where the list command has been made to only list local packages, and the `-l` option is no longer required, or supported.  If this VM is going to be using v2.0.0 of Chocolatey, I would suggest that you remove the `-l` option.

Reply:
Neat, thanks for flagging this here!